<launch>

    <arg name="project" default="dyna_ins_remover"/>
    <rosparam command="load" file="$(find dyna_ins_remover)/config/run_kitti_gt.yaml" />
    <node pkg="$(arg project)" type="gt_publisher_node" name="data_publisher_node" output="screen" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find dyna_ins_remover)/rviz/run_kitti_gt.rviz" />

</launch>
